adaway-linux.sh

```diff
@@ -52,7 +52,7 @@ mkdir -p "${TMPDIR}"
 
 # add domains from hosts-server listet in hostssources.lst
 while read src; do
-    if [[ "${src}" != "#*" ]] ; then
+    if [[ "${src}" != "#"* ]] ; then
```

Comment:
What does this change do?

Reply:
The asterisks loses its function as a wildcard when it's between apostrophes.